### PR TITLE
888835-Prepared UG documentation for MinimumPrefixCharacters and LoadMore features.

### PR DESCRIPTION
**Pull request by SivaranjithN**
Hi Team,

Added the UG documentation for MinimumPrefixCharacters and LoadMore features in SfAutocomplete and SfComboBox in .NET MAUI. Kindly review and merge the document.

Regards,
Sivaranjith N.

### CODE REVIEW

File: MAUI/Autocomplete/LoadMore.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+title: LoadMore in .NET MAUI Autocomplete control | Syncfusion
+description: Learn all about LoadMore support in Syncfusion .NET MAUI Autocomplete (SfAutocomplete) control and more here.
+platform: maui
+control: SfAutocomplete
+documentation: ug
+---
+
+# LoadMore in .NET MAUI Autocomplete (SfAutocomplete)
+
+Restrict the number of suggestions displayed and have the remaining items loaded by selecting LoadMore.We can restrict maximum suggestion to be displayed with the `MaximumSuggestion` property.
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/LoadMore.md:12]: LoadMore.We -> space missing

**Review by Choza-rajan — APPROVED** [review]

File: MAUI/Autocomplete/LoadMore.md (hunk 1 above)
Comment (SivaranjithN) [reply MAUI/Autocomplete/LoadMore.md]: Added the space.

File: maui-toc.html [hunk 2]
```diff
@@ -171,6 +171,7 @@
             <li><a href="/maui/Autocomplete/AutoSizing">AutoSizing</a></li>
             <li><a href="/maui/Autocomplete/Searching-Filtering">Searching and Filtering</a></li>
             <li><a href="/maui/Autocomplete/No-Results-Found">No Results Found</a></li>
+			<li><a href="/maui/Autocomplete/LoadMore">LoadMore</a></li>
```
Comment (Choza-rajan) [inline maui-toc.html:174]: use Maximum display item with Expander

File: maui-toc.html [hunk 3]
```diff
@@ -461,6 +462,7 @@
             <li><a href="/maui/ComboBox/Searching">Searching</a></li>
             <li><a href="/maui/ComboBox/Filtering">Filtering</a></li>
             <li><a href="/maui/ComboBox/No-Results-Found">No Results Found</a></li>
+			<li><a href="/maui/ComboBox/LoadMore">LoadMore</a></li>
```
Comment (Choza-rajan) [inline maui-toc.html:465]: use   LoadMore -> Maximum display item with Expander

Comment (SivaranjithN) [reply maui-toc.html]: Changed the name as mentioned.

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**